source-code/core/module/src/api.ts

```diff
@@ -34,8 +34,16 @@ export type ResolvedModulesFunction = (args: {
 	env: InlangEnvironment
 }) => Promise<{
 	data: {
-		resolvedPlugins: Awaited<ReturnType<ResolvePlugins>>
+		resolvedPlugins: Awaited<ReturnType<ResolvePluginsFunction>>
 		resolvedLintRules: LintRule[]
 	}
 	errors: Array<ModuleError | ModuleImportError>
 }>
+
+/**
+ * The API after resolving the modules.
+ */
+export type ResolvedModulesApi = {
+	resolvedPlugins: Awaited<ReturnType<ResolvePluginsFunction>>["data"]["plugins"]
```

Comment:
@felixhaeberle why not return the return value of the resolve plugins function as we discussed?

Reply:
What is `ResolvedModulesApi` for anyways? This is not an API?